content/developer/howtos/upgrade/upgrade_scripts.rst

```diff
@@ -99,12 +100,12 @@ Running and testing upgrade scripts
 
 .. tabs::
 
-   .. group-tab:: Odoo Online
+   .. tab:: Odoo Online
```

Comment:
Hey @Feyensv 
Thanks for your improvements! :smiley: 

I only have a question, is it possible to keep the `group-tab` in this page?

It's not used anywhere else in the same file, but if you come from the [Upgrade documentation](https://www.odoo.com/documentation/17.0/administration/upgrade.html) or go to it with the link in this page, it keeps the same tab open, allowing better continuity in the reading.

Reply:
Oh, ok then, I didn't know it would propagate between pages so I thought you only c/p it from another page without considering the use of standard tabs :D.
You can bring back that part :ok_hand: